### ISSUE TITLE
Display Bug

### ISSUE THREAD

**Issue by ttyler1999** [Nathan1258/Menubar-Weather#21]
The days in the 3 day view are off by a day. My text in red shows what they should be. It looks like they are one day behind.

I am on a Macbook Air M3 running macOS 14.6.1. The version of the app in the Finder is 2024.2 (26), which is a little wierd as the latest release possted here is 2024.1.

Thanks a lot for your help!

<img width="386" alt="Screenshot 2024-08-28 at 4 34 06 PM" src="https://github.com/user-attachments/assets/9e868f54-0d7a-4a37-8717-b726a90405df">

**Comment by Nathan1258** [comment]
Hi Ttyler1999,

Thanks for bringing it to my attention, I shall take a look at the issue today! It seems the background scheduler has not worked properly. 

Nathan

**Comment by ttyler1999** [comment]
Yes, the times do appear to be correct. Restarting the app or even my laptop does not fix the issue with the days.

Thanks!
